src/switch_exporter/server.py

```diff
@@ -65,8 +66,8 @@ def get_arguments() -> argparse.Namespace:
         '--password', default='monitor',
         help='Password on switches')
     parser.add_argument(
-        '--keyfile',
-        help='SSH key for switches')
+        '--keyfile', default=None,
```

Comment:
Do you mean `default=()`? Specifying `default=None` has no effect (it is already the default default).

Reply:
I'm not clear on the desired behavior. With `Default=None` if a user doesn't supply  `--keyfile` then password authentication is used, which is the current behavior of `master` AFAIK. From [the docs](https://asyncssh.readthedocs.io/en/latest/api.html#asyncssh.SSHClientConnectionOptions):

```
If this argument is explicitly set to [None](https://docs.python.org/3/library/constants.html#None), client public key authentication will not be performed.
```

Reply:
I see that argparser defaults to `None` though. So the addition is redundant

Reply:
This patch as a whole is changing the default behaviour. On master, SSH key authentication works if your key is in one of the standard locations that asyncssh searches e.g. `~/.ssh/id_rsa`. On your branch, that will stop working, and you have to explicitly specify `--keyfile`. I would like the behaviour when `--keyfile` is omitted to be the same as on master.

Change this argument to `default=()` to fix it.